docs/sections/calculations/index.rst

```diff
@@ -1,12 +1,12 @@
-Calculation processes
-=====================
+Processes
```

Comment:
```suggestion
Running calculations
```

I thought we were going to name the section "Running computations"/"Running calculations", which then would also include running workflows? "Processes" sounds a bit _too_ general to me. :)

Reply:
Maybe "Running processes"? I'm a bit split on the computations/calculations/processes conundrum.

Reply:
So maybe this is my obsessive streak coming out, but I changed this to just "Processes" since it was more in line with the style of the rest of the sections (single noun: Fundamentals, Data, Workflows, Plugins). Personally I would even prefer to change the style of all: Fundamentals can remain as such, but then add a pres cont verb: Running processes / Managing data / Writing workflows / Creating plugins (I like to have it in all for symmetry, but at least having "Running processes" / "Writing workflows" helps differentiate what you will learn in each). I was going to propose this afterwards but since you asked here...

Anyways, I can change back to

```suggestion
Running processes
```

and wait to see what we want to do with the other section titles later.

Reply:
Hehe, sure! Let's just write this down in a separate issue perhaps, so we can discuss there and decide on the final naming of the sections. I do like your two-word (verb+noun) suggestions though!

Reply:
Du-done #373